src/session-pool.ts

```diff
@@ -193,6 +193,7 @@ const enum errors {
 }
 
 interface SessionInventory {
+  [key: string]: Any;
```

Comment:
Why do we need this?

Reply:
Because `borrow_`, `evictIdleSessions_`, and `_release` use `this._inventory[type].-` which produce this error: `Element implicitly has an 'any' type because type 'SessionInventory' has no index signature.`, so I added a general index signature 👍

Reply:
Friendly ping! Any thoughts about this @joecervino?